ramls/scheduled-notice.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
```

Comment:
This should also have the UUID validation regular expression

Reply:
added the expression